helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -362,7 +376,7 @@ public void addDeploymentGroup(final DeploymentGroup deploymentGroup)
   public DeploymentGroup getDeploymentGroup(final String name)
       throws DeploymentGroupDoesNotExistException {
     log.debug("getting deployment-group: {}", name);
-    final ZooKeeperClient client = provider.get("getDeploymentGroup");
+    final ZooKeeperClient client = provider.get("getDeploymentGroupResponse");
```

Comment:
Perhaps keep this as before for easier debugging, ie have it match the method name?